app/views/works/show.html.erb

```diff
@@ -1,267 +1,232 @@
 <div class="container" id="work">
-  <div class="row">
-    <div class="col">
-      <header>
-        <span class="header-text"><%= @work.title %></span>
-        <%= link_to edit_work_path(@work), aria: { label: "Edit #{@work.title}"} do %><span class="fas fa-pencil-alt edit"></span><% end %>
-        <% if @work.state != 'deposited' %><span class="font-italic" style="margin-left: 30px;">Draft - Not deposited</span><% end %>
-      </header>
-    </div>
-  </div>
+  <%= render Works::ApprovalComponent.new(work: @work) %>
 
-  <div class="row">
-    <div class="col">
-      <table class="table table-sm">
-        <thead class="table-light">
-        <tr>
-          <th scope="col" class="table-title col-3">Details</th>
-          <th scope="col"></th>
-        </tr>
-        </thead>
-        <tbody>
-        <% if @work.purl %>
+  <section class="details">
+    <header class="title">
+      <span class="header-text"><%= @work.title %></span>
+      <%= link_to edit_work_path(@work), aria: { label: "Edit #{@work.title}"} do %><span class="fas fa-pencil-alt edit"></span><% end %>
+      <% if @work.state != 'deposited' %><span class="font-italic" style="margin-left: 30px;">Draft - Not deposited</span><% end %>
```

Comment:
I think you can do `unless @work.deposited?` here too if you're cool with it. Yay for magic state machine methods.

Reply:
Sure, this isn't really code I changed though (other than indent level).

Reply:
That'll teach you to indent.